### PR TITLE
mds/MDSRank.cc: remove unneeded searching of map

### PR DESCRIPTION
**Pull request by stiopaa1**
Signed-off-by: Michal Jarzabek <stiopa@gmail.com>

### CODE REVIEW

File: src/mds/MDSRank.cc [hunk 1]
```diff
@@ -753,10 +753,13 @@ void MDSRank::send_message_mds(Message *m, mds_rank_t mds)
   }
 
   // send mdsmap first?
-  if (mds != whoami && peer_mdsmap_epoch[mds] < mdsmap->get_epoch()) {
+  auto peer_mdsmap_epoch_entry =
+    peer_mdsmap_epoch.emplace(mds, version_t()).first;
+
+  if (mds != whoami && peer_mdsmap_epoch_entry->second < mdsmap->get_epoch()) {
```
Comment (liewegas) [inline src/mds/MDSRank.cc]: This is different in that we insert a peer_mdsmap_epoch entry when mds == whoami

**Comment by dachary** [comment]
jenkins test this please (no logs)

**Comment by jcsp** [comment]
@stiopaa1 are you coming back to this?  See Sage's comment above.

**Comment by jcsp** [comment]
This seems to have gone stale -- because this is just a cleanup I'm going to go ahead and close it.  Feel free to reopen a PR if you want to submit a revised version of the patch.

**Pull request closed without merging**